docs/how-to-work-on-the-component-library.md

```diff
@@ -92,6 +92,10 @@ To run tests against the component library, run the following command from the r
 pnpm run test-ui-components
 ```
 
+## Proposing a Pull Request (PR)
+
+After you've committed your changes, check here for [how to open a Pull Request](how-to-open-a-pull-request.md).
```

Comment:
```suggestion
After you've committed your changes, check here [how to open a Pull Request](how-to-open-a-pull-request.md).
```

Reply:
I think I like the `for` in there because it makes the link part of a complete sentence.

Reply:
I'm in agreement with Jeremy - the current structure is grammatically correct.

Reply:
Feel free to revert it back